docs/source/markdown/podman-systemd.unit.5.md

```diff
@@ -0,0 +1,325 @@
+% podman-systemd.unit 5
+
+## NAME
+
+podman\-systemd.unit - systemd units using podman
+
+## SYNOPSIS
+
+*name*.container, *name*.volume
+
+### Podman unit search path
+
+ * /etc/containers/systemd/
+ * /usr/share/containers/systemd/
+
+### Podman user unit search path
+
+ * $XDG_CONFIG_HOME/containers/systemd/
+ * ~/.config/containers/systemd/
+
+## DESCRIPTION
+
+Podman supports starting containers (and creating volumes) via systemd by using a
+[systemd generator](https://www.freedesktop.org/software/systemd/man/systemd.generator.html).
+These files are read during boot (and when `systemctl daemon-reload` is run) and generate
+corresponding regular systemd service unit files. Both system and user systemd units are supported.
+
+The podman generator reads the search paths above and reads files with the extensions `.container`
+and `.volume`, and for each file generates a similarly named `.service` file. These units can be started
+and managed with systemctl like any other systemd services.
+
+The podman files use the same format as [regular systemd unit files](https://www.freedesktop.org/software/systemd/man/systemd.syntax.html).
+Each file type has a custom section (for example `[Container]`) that is handled by podman, and all
+other sections will be passed on untouched. This means you can use any normal systemd configuration
```

Comment:
Do not add `you` in man page,  Perhaps

IE any normal systemd configuration like dependencies or cgroup limits.